package/plugin-gargoyle-tor/files/www/tor.sh

```diff
@@ -18,140 +18,181 @@
+					<br/>
+					<em><span id="mode_description"></span></em>
+					<br/>
+				</div>
+
+				<div id='tor_relay_port_container' class="form-control">
+					<label for='tor_relay_port' id='tor_relay_port_label'><%~ BrRPrt %>:</label>
+					<input type="text" class="form-control" id='tor_relay_port' size='9' onkeyup='proofreadPort(this)' />
+				</div>
+
+				<div id='tor_obfsproxy_port_container' class="form-control">
+					<label for='tor_obfsproxy_port' id='tor_obfsproxy_port_label'><%~ ObfPrt %>:</label>
+					<input type="text" class="form-control" id='tor_obfsproxy_port' size='9' onkeyup='proofreadPort(this)' />
+				</div>
+
+				<div id='tor_relay_max_bw_container' class="form-control">
+					<label for='tor_relay_max_bw' id='tor_relay_max_bw_label'><%~ MaxRB %>:</label>
+					<span><input type="text" class="form-control" id='tor_relay_max_bw' size='9' onkeyup='proofreadNumeric(this)' /><em>&nbsp;&nbsp;<%~ KBs %></em></span>
+				</div>
+
+				<div id='tor_relay_publish_container' class="form-control">
+					<label for='tor_relay_publish' id='tor_relay_publish_label'><%~ PubBrDB %>:</label>
+					<select class="form-control" id="tor_relay_publish">
+						<option value="1"><%~ PubBr %></option>
+						<option value="0"><%~ NoPub %></option>
+					</select>
+				</div>
+
+				<div id='tor_relay_nickname_container' class="form-control">
+					<label for='tor_relay_nickname' id='tor_relay_nickname_label'><%~ Nick %>:</label>
+					<input type="text" class="form-control" id='tor_relay_nickname' />
+				</div>
+
+				<div id='tor_relay_contact_container' class="form-control">
+					<label for='tor_relay_contact' id='tor_relay_contact_label'><%~ Mail %>:</label>
+					<textarea class="form-control" id='tor_relay_contact' ></textarea>
+				</div>
+
+				<div id='tor_relay_status_link_container' class="form-control">
+					<span class='nocolum'><%~ VisMsg %> <a href="http://torstatus.blutmagie.de/"><%~ GlbMsg %></a></span>
+				</div>
 
-		<div id='tor_relay_publish_container'>
-			<label  class='wideleftcolumn' for='tor_relay_publish' id='tor_relay_publish_label'><%~ PubBrDB %>:</label>
-			<select class="rightcolumn" id="tor_relay_publish">
-				<option value="1"><%~ PubBr %></option>
-				<option value="0"><%~ NoPub %></option>
-			</select>
-		</div>
+			</div>
 
-		<div id='tor_relay_nickname_container'>
-			<label  class='wideleftcolumn' for='tor_relay_nickname' id='tor_relay_nickname_label'><%~ Nick %>:</label>
-			<input type="text" class="rightcolumn" id='tor_relay_nickname' />
 		</div>
+	</div>
 
-		<div id='tor_relay_contact_container'>
-			<label  class='wideleftcolumn' for='tor_relay_contact' id='tor_relay_contact_label'><%~ Mail %>:</label>
-			<textarea class="rightcolumn" id='tor_relay_contact' ></textarea>
-		</div>
+</div>
 
-		<div id='tor_relay_status_link_container'>
-			<span class='nocolum'><%~ VisMsg %> <a href="http://torstatus.blutmagie.de/"><%~ GlbMsg %></a></span>
-		</div>
+<div class="row tor_data_dir_section">
```

Comment:
typo here.
Tor needs work still. How do we get you able to test it on your device?